Makefile

```diff
@@ -0,0 +1,23 @@
+test: build-buffertools build-leveldown
+
+build-buffertools:
```

Comment:
This and `build-leveldown` should probably depend on an `npm install` task if you want these tests to be self-contained for CI.